src/libsyntax_expand/base.rs

```diff
@@ -1072,7 +1072,8 @@ impl<'a> ExtCtxt<'a> {
     /// This unifies the logic used for resolving `include_X!`, and `#[doc(include)]` file paths.
     ///
     /// Returns an absolute path to the file that `path` refers to.
-    pub fn resolve_path(&self, path: impl Into<PathBuf>, span: Span) -> PathBuf {
+    pub fn resolve_path(&self, path: impl Into<PathBuf>, span: Span)
+                        -> Result<PathBuf, DiagnosticBuilder<'a>> {
```

Comment:
Please use the same formatting as the method below.

Reply:
OK. Wasn't sure what style to use, so copied from this method: https://github.com/rust-lang/rust/blob/0b97ef8d5f78ea7144dbc2afb0ff3e055fb5056a/src/libsyntax_expand/base.rs#L1126-L1127